src/main/java/seedu/address/model/person/Performance.java

```diff
@@ -62,4 +62,8 @@ public boolean equals(Object other) {
     public int hashCode() {
         return value.hashCode();
     }
+
+    public int getUrgency() {
```

Comment:
I think if it is not a getter method, pls don't name it this way. Can we get rid of this method and sort student performance reversely (just a suggestion)?

Reply:
Thanks for the comment! Actually, I was intending for this to be a dummy method for now. In the future this will involve more calculations to get the urgency level of a student, but I will change the name to be ```calculateUrgency()``` instead!